src/game/entity/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+target_sources(app PRIVATE 
+dev/red_square.cpp
```

Comment:
could you add indent to be consistent with the other `CMakeList`s? :wink:

Reply:
See 0fc88612472d3e788f89144e0840b55a9e7f39e1

Reply:
:+1: